obal/data/modules/srpm.py

```diff
@@ -0,0 +1,122 @@
+"""
+Build SRPM
+"""
+
+import shutil
+import os
+import subprocess
+from tempfile import mkdtemp
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.chdir import chdir # pylint:disable=import-error,no-name-in-module
+
+def run_command(command):
+    """
+    Run a system command
+    """
+    return subprocess.check_output(
+        ' '.join(command),
+        universal_newlines=True,
+        shell=True,
```

Comment:
IMHO not needed

Reply:
See https://github.com/theforeman/obal/pull/232#discussion_r468534267